tests/config/test_data.config

```diff
@@ -0,0 +1,70 @@
+
+// Base directory for test data
+def test_data_dir = "${launchDir}/tests/data/"
+
+params {
+    test_data {
+        'sarscov2' {
+            'genome' {
+                genome.fasta                              = "${test_data_dir}/genomics/sarscov2/genome/genome.fasta"

```

Comment:
This probably needs to be quoted, otherwise the namespace will be nested further